src/nft/components/collection/CollectionStats.tsx

```diff
@@ -277,7 +277,7 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
   )
 
   return (
-    <Row gap={{ sm: '20', md: '60' }} {...props}>
+    <Row gap={{ sm: '36', md: '60' }} {...props}>
```

Comment:
is this change intentional? unrelated to analytics?

Reply:
Yes, intentional. This was a nit on a different PR and I merged that one before I saw the comment.